### PR TITLE
[7] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by kazeemerinfolami**
## Task: `[7] Develop a Contact Form Page using HTML`
- `Contact Form`
Screen Shot 2020-09-20 at 9 13 54 PM

### CODE REVIEW

**Comment by kazeemerinfolami** [comment]
Thank you for your corrections.

File: submissions/kazeemErinfolami/htmlForms/contact.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact</title>
+  </head>
+  <body>
+    <form>
+      <div>
+        <h1>Contact Form</h1>
+      </div>
+      <hr />
```
Comment (ToluAkin) [inline submissions/kazeemErinfolami/htmlForms/contact.html:13]: @kazeemerinfolami Remove the backslash here

File: submissions/kazeemErinfolami/htmlForms/contact.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      <label for="name"><b>Full Name <b>:</label>
+      <input type="text" placeholder="name" id="name" name="name" /><br><br>
```
Comment (ToluAkin) [inline submissions/kazeemErinfolami/htmlForms/contact.html:15]: Same here in the input @kazeemerinfolami

File: submissions/kazeemErinfolami/htmlForms/contact.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      <label for="email"><b>Email<b> :</label>
+      <input type="email" placeholder="email" id="email" name="email" /><br><br >
```
Comment (ToluAkin) [inline submissions/kazeemErinfolami/htmlForms/contact.html:17]: Same here in the input @kazeemerinfolami

File: submissions/kazeemErinfolami/htmlForms/contact.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+      <label for="subject"><b>Subject <b>:</label>
+      <input type="text" placeholder="Subject" id="subject" name="subject" /><br><br>
```
Comment (ToluAkin) [inline submissions/kazeemErinfolami/htmlForms/contact.html:19]: Same here in the input @kazeemerinfolami

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**